src/libraries/System.Runtime.InteropServices/tests/ComInterfaceGenerator.Unit.Tests/GeneratedComInterfaceAnalyzerTests.cs

```diff
@@ -35,8 +35,9 @@ interface IFoo
                         void Bar() {}
                     }
 
-                    public partial class MyComWrappers : GeneratedComWrappersBase<ComObject>
+                    public unsafe partial class MyComWrappers : GeneratedComWrappersBase
                     {
+                        protected override ComInterfaceEntry* ComputeVtables(object obj, CreateComInterfaceFlags flags, out int count) {count = 1; return null;}
```

Comment:
```suggestion
                        protected override ComInterfaceEntry* ComputeVtables(object obj, CreateComInterfaceFlags flags, out int count) {count = 0; return null;}
```

Unless we are trying to force a failure, returning `null` should mean a count of `0`.

Reply:
Throughout.